docs/migration/2.0.md

```diff
@@ -0,0 +1,221 @@
+## 2.0 Migration
+
+## Configuration
+
+### Beemo configuration has moved
+
+Previously, configuration was either defined in a root `configs/beemo.js` file, or a `beemo` block
+within `package.json`. Configuration must now be defined in `.config/beemo.js` (or `.json`, `.yaml`,
+etc). Package level config has been removed entirely.
+
+```js
+// .config/beemo.js
+module.exports = {
+  module: '<config-module>',
+};
+```
+
+### Driver overrides have been removed from `package.json`
+
+The ability to configure drivers in a `beemo.<driver>` block within `package.json` has been removed.
+Instead, configure the driver at `.config/beemo/<driver>.js` (or `.json`, `.yaml`, etc).
+
+```json
+// Before - package.json
+{
+  "beemo": {
+    "eslint": {
+      "rules": {
+        "no-console": "off"
+      }
+    }
+  }
+}
+```
+
+```js
+// After - .config/beemo/eslint.js
+module.exports = {
+  rules: {
+    'no-console': 'off',
+  },
+};
+```
+
+### Drivers configuration structure has changed
+
+When configuring drivers with the `drivers` setting, either supply a list of names.
+
+```js
+// .config/beemo.js
+module.exports = {
+  module: '<config-module>',
+  drivers: ['babel', 'jest'],
+};
+```
+
+Or an object of names that map to booleans (enable or disable the driver), or an options object.
+
+```js
+// .config/beemo.js
+module.exports = {
+  module: '<config-module>',
+  drivers: {
+    babel: true,
+    jest: {
+      env: {
+        NODE_ENV: 'test',
+      },
+    },
+  },
+};
+```
+
+The old format of mixing strings and objects within a list is no longer supported.
+
+### Driver and script modules must export a factory function and have a name
+
+If you're using custom driver and script modules, they must now default export a function that
+returns a class instance, instead of exporting a class declaration. Furthermore, all driver and
+script instances must have a `name` property (which is the name of the NPM module).
+
+```ts
+// Before
+export default class CustomDriver extends Driver {}
+```
+
+```ts
+// After
+class CustomDriver extends Driver {
+  name = 'npm-module-name-driver';
+}
+
+export default (options) => new CustomDriver(options);
+```
+
+## Beemo
+
+### Logging has been removed
+
+All logging methods have been removed. Use the native console instead.
+
+```ts
+// Before
+beemo.log();
+beemo.log.error();
+beemo.console.log();
+
+// After
+console.log();
+console.error();
```

Comment:
Does this not get swallowed by the rewrites done by the react output? Or is this all running in child processes that get aggregated?

Reply:
The console logs are captured and output above the continuous React rendered output.